lib/nx_zlib.c

```diff
@@ -156,7 +156,7 @@ void *nx_alloc_buffer(uint32_t len, long alignment, int lock)
 	ts = nx_get_time();
 #endif
 
-	buf = malloc( len + alignment + sizeof(nx_alloc_header_t) );
+	buf = calloc( 1, len + alignment + sizeof(nx_alloc_header_t) );
```

Comment:
@lucmaga could you elaborate more on where is the jump depending on uninitialized value, please?

Reply:
@lucmaga how are you executing the test? Is test/valgrind.supp being used like explained in commit https://github.com/libnxz/power-gzip/commit/13712cd76d505f3d49a9a289dccd7d05789ea2f1?

If you already are, I think this is another false-positive because valgrind is not able to know the accelerator wrote to that address and we might need another suppresion in test/valgrind.supp.

Reply:
Right. It is indeed a false-positive that is fixed by the suppression.
I remove this commit.

Reply:
Not sure if I'm continue to have issues with this PR history but I still see this commit in the new series.

Reply:
Sorry, It was my mistake. Now is correct.